feature-libs/product-configurator/rulebased/styles/_configurator-overview-form.scss

```diff
@@ -15,127 +15,314 @@
-        padding-inline-start: 16px;
-        padding-inline-end: 16px;
+        padding-block-start: 20px;
+        padding-block-end: 20px;
+        margin-inline-start: -20px;
+        margin-inline-end: -20px;
       }
 
-      &.general {
-        @include media-breakpoint-up(md) {
-          &:not(:first-of-type) {
-            .cx-attribute-label {
-              visibility: hidden;
-            }
+      &.topLevel {
+        h2 {
+          padding-inline-start: 32px;
+          padding-inline-end: 32px;
+          padding-block-start: 16px;
+          padding-block-end: 16px;
+
+          @include media-breakpoint-down(sm) {
+            padding-inline-start: 16px;
+            padding-inline-end: 16px;
+            padding-block-start: 16px;
+            padding-block-end: 16px;
+          }
+
+          font-size: 1.25rem;
+          font-weight: 700;
+          border-bottom: solid 1px var(--cx-color-light);
+          border-top: solid 1px var(--cx-color-light);
+          border-left-style: none;
+          border-right-style: none;
+          background-color: var(--cx-color-inverse);
+          text-transform: none;
+          position: sticky;
+          top: 2px;
+          z-index: -1;
+          span {
+            @include cx-configurator-truncate-content();
           }
         }
+      }
+
+      &.subgroupTopLevel {
+        margin-bottom: -20px;
+      }
 
+      &.subgroup {
         @include media-breakpoint-down(sm) {
-          .cx-attribute-label {
-            display: none;
+          padding-block-start: 20px;
+          padding-block-end: 20px;
+          margin-inline-start: 0px;
+          margin-inline-end: 0px;
+        }
+
+        h2 {
+          padding-inline-start: 32px;
+          padding-inline-end: 32px;
+          padding-block-start: 16px;
+          padding-block-end: 16px;
+
+          @include media-breakpoint-down(sm) {
+            padding-inline-start: 16px;
+            padding-inline-end: 16px;
+            padding-block-start: 16px;
+            padding-block-end: 16px;
           }
 
-          &.last-value-pair {
-            .cx-attribute-label {
-              display: inline;
-            }
+          border: solid 1px var(--cx-color-light);
+          background-color: var(--cx-color-background);
+          font-size: 1rem;
+          font-weight: normal;
+          text-transform: uppercase;
+          position: sticky;
+
+          span {
+            @include cx-configurator-truncate-content();
           }
         }
       }
 
-      &.bundle {
-        background-color: var(--cx-color-background);
+      &.subgroupLevel2 {
+        h2 {
+          top: 60px;
+          z-index: -2;
+        }
+      }
+
+      &.subgroupLevel3 {
+        h2 {
+          top: 112px;
+          z-index: -3;
+        }
+      }
 
-        @include media-breakpoint-up(md) {
-          &:not(:first-of-type) {
+      &.subgroupLevel4 {
+        h2 {
+          top: 164px;
```

Comment:
Can we use variables for this?
Does this work, when I change the default font size of my device?

Reply:
I will introduce variables.
Yes, it als works if the size is changed
<img width="1121" alt="Screenshot 2022-07-29 at 09 12 58" src="https://user-images.githubusercontent.com/54309425/181703782-a4f9c455-e88c-4ffb-8987-ba73fae78b53.png">
.

Reply:
Done